requirements.txt

```diff
@@ -42,5 +42,6 @@ bandit==1.7.0
 git-repo==1.10.3
 pywinrm>=0.4.2 ; platform_system == "Linux" or platform_system=='Windows'
 wmi>=1.5.1; platform_system=='Windows'
+deepdiff
```

Comment:
Add version and system pls

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2437/commits/7955e10644d7e0d5c4e2db1e82d28b85e367946a